code/modules/admin/tabs/admin_tab.dm

```diff
@@ -796,3 +796,21 @@
 
 	SSticker.mode.toggleable_flags ^= MODE_SHIPSIDE_SD
 	message_admins("[src] has [MODE_HAS_TOGGLEABLE_FLAG(MODE_SHIPSIDE_SD) ? "toggled SD protection off, Yautja can now big self destruct anywhere" : "toggled SD protection on, Yautja can now only big self destruct on the hunting grounds"].")
+
+/client/proc/toggle_hardcore_perma()
+	set name = "Toggle Hardcore"
+	set category = "Admin.Flags"
+
+	if(!admin_holder || !check_rights(R_MOD, FALSE))
+		return
+
+	if(!SSticker.mode)
+		to_chat(usr, SPAN_WARNING("A mode hasn't been selected yet!"))
+
+	if(!MODE_HAS_TOGGLEABLE_FLAG(MODE_HARDCORE_PERMA))
+		if(alert("Are you sure you want to toggle Hardcore mode on? This will cause all humans to instantly go perma on death.",,"Yes","Cancel") != "Yes")
```

Comment:
```suggestion
		if(!MODE_HAS_TOGGLEABLE_FLAG(MODE_HARDCORE_PERMA) && tgui_alert(usr, "Are you sure you want to toggle Hardcore mode on? This will cause all humans to instantly go perma on death.", "Confirmation", list("Yes", "Cancel")) != "Yes")
```
Alternatively change the string like message_admins below does if you want to confirm disable too.